src/mixins/allowable-actions-mixin.ts

```diff
@@ -127,7 +127,8 @@ export default class AllowableActionsMixin extends Vue {
       }
 
       case AllowableActions.PUT_BACK_ON: {
-        return this.isAllowedFiling(FilingTypes.PUT_BACK_ON)
+        return this.isAllowedFiling(FilingTypes.PUT_BACK_ON) &&
+        !!GetFeatureFlag('supported-put-back-on-entities')?.includes(this.getLegalType)
```

Comment:
When do you get if `GetFeatureFlag('supported-put-back-on-entities')` is falsey? I'm wondering if the `!!` at the beginning of the line changes line 131 to True? (which would be incorrect, since we don't want to display a feature that we can't be sure is enabled)

Reply:
I was just going for consistency because the code that checks feature flags are already using !!

Reply:
The way I see it though, it seems like a small price(1 character diff) for odd "just-in-case" scenarios

Reply:
I'm not blaming you. I'm just having a critical look at the code. And yes, consistency is very good (even if it's wrong, because then everything is wrong and can be fixed together instead of separately).

But actually, I'm looking at this again and I think I must have had crossed eyes for a moment...

If GetFeatureFlag is null/undefined then the inner expression is undefined, but we do a `!!` on it (not a single !), so that will evaluate to False, which is what we want.

It's good as is. Carry on.